Test/Unit/Logger/AlgoliaLoggerTest.php

```diff
@@ -19,7 +19,7 @@ protected function setUp(): void
         $this->systemLoggerHandler = $this->createMock(SystemLoggerHandler::class);
         $this->algoliaLoggerHandler = $this->createMock(AlgoliaLoggerHandler::class);
 
-        $this->algoliaLogger = new Logger(
+        $this->algoliaLogger = new AlgoliaLogger(
```

Comment:
This is passing, but I still need to set the class attributes as nullable to make it work 😅

Reply:
Ah! Yes I need to remember to make all of these nullable going forward. I've gone and tweaked the unit test for this.